package.json

```diff
@@ -124,6 +124,11 @@
                     "default": true,
                     "description": "Always rank completion items on the server as you type. This produces more accurate results at the cost of higher latency than client-side filtering."
                 },
+                "clangd.maxRestartCount": {
```

Comment:
as discussed offline...

I think from a UX perspective "do you want to restart" is a much easier question to answer, and being able to set the count to 2 or 99 isn't actually very important.

So I'd make this "restartAfterCrash", boolean, default true, and the logic could be restartAfterCrash ? 4 : 0 or something like that.

Reply:
Thanks, done.